apps/hostel/schemas.py

```diff
@@ -14,5 +16,5 @@ class HostelListOut(BaseOutSchema):
 
 
 class BedPlaceOut(BaseOutSchema):
-    bed_place_id: int
+    bed_place_id: BedPlaceItem
     bed_place_name: str
```

Comment:
It seems to me it should be 'bed_place_name' type to be changed instead of 'bed_place_id' type. What do you think?

Reply:
Make sense